packages/widget/src/pages/SwapExecutionPage/SwapExecutionPageRouteDetailed.tsx

```diff
@@ -87,12 +87,19 @@ export const SwapExecutionPageRouteDetailed = ({
     return _onClickEditDestinationWallet;
   }, [isSignRequired, chainAddresses, swapExecutionState, _onClickEditDestinationWallet]);
 
+  // special case needed for axelar transfers where the source denom is not the first operation denom
+  const ibcTransferToAxelarOp = operations.find(op => op.ibcTransferToAxelar);
```

Comment:
this can be wrong, if axelarTransfer not in the first operation index it will be wrong